### PR TITLE
fix: add --project flag to create theme for consistency

### PR DESCRIPTION
**Pull request by hmble**
Signed-off-by: Samyak Bakliwal <w3bcode@gmail.com>

### CODE REVIEW

File: cli/create.go [hunk 1]
```diff
@@ -45,17 +45,20 @@ func newCreateProjectCmd() *cobra.Command {
...
-			path := args[0]
-			name := args[1]
+			name := args[0]
 
-			return core.CreateTheme(path, name)
+			return core.CreateTheme(options, name)
 		},
 	}
 
+	createThemeCmd.Flags().StringVar(&options.Project, "project", ".", `project path to create new theme in.`)
```
Comment (aligator) [inline cli/create.go:62]: what do you think about adding a short alias "-p"?

File: cli/create.go [hunk 2]
```diff
@@ -45,17 +45,20 @@ func newCreateProjectCmd() *cobra.Command {
 
 // newCreateThemeCmd creates the `verless create theme` command.
 func newCreateThemeCmd() *cobra.Command {
+	var (
+		options core.CreateThemeOptions
+	)
 	createThemeCmd := cobra.Command{
 		Use:   "theme PROJECT NAME",
```
Comment (aligator) [inline cli/create.go:52]: not a problem from your PR, but shouldn't that be like this?
```suggestion
		Use:   "theme THEME_NAME",
```

**Review by aligator — CHANGES_REQUESTED** [review]
Looks good and works, I just have two small requests.

**Comment by hmble** [comment]
I guess I should also add `-p` shorthand to PR #182

**Pull request merged**